core/org.openjdk.jmc.agent/.project

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you want to add `.project` and `.settings/org.eclipse.jdt.core.prefs` back to the vcs in the <root>/agent directory?